readme.md

```diff
@@ -172,7 +172,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [nginx](https://github.com/fcambus/nginx-resources)
 - [Dropwizard](https://github.com/stve/awesome-dropwizard)
 - [Kubernetes](https://github.com/ramitsurana/awesome-kubernetes)
-
+- [Vorpal](https://github.com/vorpaljs/awesome-vorpaljs)
```

Comment:
True. I used to call it Vorpal.js, but have since just changed the name to Vorpal. I've changed the repo name (org name has to stay the same, `vorpal` is taken) to Vorpal now.

I guess I should throw it in Misc? Was trying to avoid that but I don't see a better fit.

Thanks! :sparkles: